nodes/recipe_handler.py

```diff
@@ -71,14 +71,16 @@ def trace(msg, *args):
 
 
 #------------------------------------------------------------------------------
+# Returns the current 'state', an unordered dictionary of the current value
```

Comment:
This should be part of the docstring for the function below. Keeping function documentation in docstrings allows them to be accessed computationally, so you can do things like

- generate documentation from comments
- automatically run tests in comments
- `help(func_name)` .